core/audits/redirects.js

```diff
@@ -40,12 +40,13 @@ class Redirects extends Audit {
    * This method generates the document request chain including client-side and server-side redirects.
    *
    * Example:
-   *    GET /initialUrl => 302 /firstRedirect
+   *    GET /requestedUrl => 302 /firstRedirect
    *    GET /firstRedirect => 200 /firstRedirect, window.location = '/secondRedirect'
    *    GET /secondRedirect => 302 /thirdRedirect
-   *    GET /thirdRedirect => 200 /mainDocumentUrl
+   *    GET /thirdRedirect => 302 /mainDocumentUrl
+   *    GET /mainDocumentUrl => 200 /mainDocumentUrl
```

Comment:
not that important, but this didn't quite make sense (there had to be one last redirect)